### PR TITLE
Dependency and workflow updates

### PR DESCRIPTION
**Pull request by BrandtH22**
- replaces yarn with npm in deployment
- adds review deployment and deletion
- updates runnable import to referential path rather than absolute for ease of translation integrations
- adds trailing slash for seo
- adds Chia Docs to the header
- updates all dependencies, imports, and readme for running

### CODE REVIEW

File: k8s/certificate.yaml
Comment (Starttoaster) [inline k8s/certificate.yaml:11]:
```suggestion
    - "*.chialisp-web.chia.net"
```
Same reason

File: k8s/review.yaml.j2
Comment (Starttoaster) [inline k8s/review.yaml.j2:13]:
```suggestion
    - host: {{ REVIEW_SLUG }}.chialisp-web.chia.net
```
Same reason

File: .github/workflows/deploy-review.yml
Comment (Starttoaster) [inline .github/workflows/deploy-review.yml:87]: Need a Dockerfile at the root of this repo

Comment (Starttoaster) [reply .github/workflows/deploy-review.yml:87]: The one from chia-docs should just work since we build the app outside of the Dockerfile first:
```
FROM nginx:stable-alpine
COPY build /usr/share/nginx/html
EXPOSE 80
CMD nginx -g "daemon off;"
```

File: k8s/certificate.yaml
Comment (Starttoaster) [inline k8s/certificate.yaml:11]:
```suggestion
    - "*.chialisp-web.chia.net"
```

**Review by Starttoaster — APPROVED** [review]
My approval is for the CI/Docker/K8s manifest portions of the diff. I would gather additional reviewer(s) for the docs content where applicable. If future changes to docs content dismiss my review, that's fine. Just wanted to document my approval of that portion since github doesn't have a foolproof method of split approvals for PRs that do multiple things.

**Review by Rigidity — APPROVED** [review]
Looks good to me on the docs side of things

**Pull request merged**
